### PR TITLE
[native] Add e2e test for chi_squared_cdf function

### PR DESCRIPTION
**Pull request by Krishna-Prasad-P-V**
## Test Plan
Test Plan -  Added e2e test for new scalar functions chi_sqaured_cdf.
PR for chi_squared_cdf - https://github.com/facebookincubator/velox/pull/5302

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java [hunk 1]
```diff
@@ -49,4 +51,10 @@ public void testBetaCDF()
     {
         assertQuery("SELECT beta_cdf(nationkey, 0.2, 0.2) from nation where nationkey > 0");
     }
+
+    @Test
+    public void testChiSquaredCDF()
+    {
+        assertQuery("SELECT chi_squared_cdf(quantity, tax) FROM lineitem WHERE quantity > 0.0 AND tax >= 0.0");
```
[the remark this answers is not in the document]
Comment (Krishna-Prasad-P-V) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java:58]: I wanted a table with 2 double columns which could be used as inputs to this function. I will find another table which is not this big.

[the remark this answers is not in the document]
Comment (Krishna-Prasad-P-V) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java]: @aditi-pandit - Changed the query to use another table. Also, confirmed that with schema tpch.sf1, the query would generate 472 rows only. Added two queries for the sake of testing.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java [hunk 2]
```diff
@@ -27,6 +28,7 @@ protected void createTables()
     {
         QueryRunner queryRunner = (QueryRunner) getExpectedQueryRunner();
         createNation(queryRunner);
+        createLineitem(queryRunner);
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java:31]: Is this needed anymore ?

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java [hunk 3]
```diff
@@ -49,4 +51,11 @@ public void testBetaCDF()
     {
         assertQuery("SELECT beta_cdf(nationkey, 0.2, 0.2) from nation where nationkey > 0");
     }
+
+    @Test
+    public void testChiSquaredCDF()
+    {
+        assertQuery("SELECT chi_squared_cdf(acctbal, 2.0) FROM supplier WHERE acctbal > 0.0 AND acctbal < 500.0");
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java:58]: It didn't seem like you created the Supplier table above. Please can you recheck.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java (hunk 2 above)
Comment (Krishna-Prasad-P-V) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java]: Fixed.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java (hunk 3 above)
Comment (Krishna-Prasad-P-V) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeProbabilityFunctionQueries.java:58]: Fixed.

**Review by aditi-pandit — APPROVED** [review]

**Pull request merged**